src/main/java/com/minecraftabnormals/upgrade_aquatic/core/mixin/DrownedModelMixin.java

```diff
@@ -0,0 +1,68 @@
+                this.bipedLeftArm.rotateAngleX = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleX, ((float)Math.PI / 2F) * multiplier1);
+                this.bipedRightArm.rotateAngleX = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleX, ((float)Math.PI / 2F) * multiplier1);
+                this.bipedLeftArm.rotateAngleY = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleY, (float)Math.PI);
+                this.bipedRightArm.rotateAngleY = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleY, (float)Math.PI);
+                this.bipedLeftArm.rotateAngleZ = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleZ, 5.012389F - 1.8707964F * multiplier1);
+                this.bipedRightArm.rotateAngleZ = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleZ, 1.2707963F + 1.8707964F * multiplier1);
+            } else if (limbSwingRemainder >= 22.0F && limbSwingRemainder < 26.0F) {
+                float multiplier2 = (limbSwingRemainder - 22.0F) / 4.0F;
+                this.bipedLeftArm.rotateAngleX = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleX, ((float)Math.PI / 2F) - ((float)Math.PI / 2F) * multiplier2);
+                this.bipedRightArm.rotateAngleX = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleX, ((float)Math.PI / 2F) - ((float)Math.PI / 2F) * multiplier2);
+                this.bipedLeftArm.rotateAngleY = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleY, (float)Math.PI);
+                this.bipedRightArm.rotateAngleY = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleY, (float)Math.PI);
+                this.bipedLeftArm.rotateAngleZ = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleZ, (float)Math.PI);
+                this.bipedRightArm.rotateAngleZ = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleZ, (float)Math.PI);
+             } if (entityIn.isActualySwimming()) {
+                this.bipedHead.rotateAngleX = this.rotLerpRad(this.swimAnimation, this.bipedHead.rotateAngleX, (-(float)Math.PI / 4F));
+             } else {
+                this.bipedHead.rotateAngleX = this.rotLerpRad(this.swimAnimation, this.bipedHead.rotateAngleX, headPitch * ((float)Math.PI / 180F));
+             }
+             this.bipedLeftLeg.rotateAngleX = MathHelper.lerp(this.swimAnimation, this.bipedLeftLeg.rotateAngleX, 0.3F * MathHelper.cos(limbSwing * 0.33333334F + (float)Math.PI));
+             this.bipedRightLeg.rotateAngleX = MathHelper.lerp(this.swimAnimation, this.bipedRightLeg.rotateAngleX, 0.3F * MathHelper.cos(limbSwing * 0.33333334F));
+             } else {
+                entityIn.setPose(Pose.STANDING);
+             }
+    }
+
+    private float getArmAngleSq(float limbSwing) {
+        return -65.0F * limbSwing + limbSwing * limbSwing;
+    }
```

Comment:
Again, add a blank line after this method to be consistent with the blank line at the top of the class.